userdatamodel/user.py

```diff
@@ -53,6 +53,12 @@ class User(Base):
     idp_id = Column(Integer, ForeignKey('identity_provider.id'))
     identity_provider = relationship('IdentityProvider', backref='users')
 
+    google_proxy_group_id = Column(
+        String(90), ForeignKey('google_proxy_group.id'))
+
+    google_proxy_group = relationship(
+        'GoogleProxyGroup', backref=backref(__tablename__, uselist=False))
```

Comment:
Should add a `cascade='all,delete-orphan'` for this relationship I think, and perhaps if a user is deleted the associated `GoogleProxyGroup` should be deleted as well?

Reply:
fixed in f3c6dd1